sql_generators/glean_usage/templates/baseline_clients_last_seen_v1.schema.yaml

```diff
@@ -66,3 +66,6 @@ fields:
 - mode: NULLABLE
   name: install_source
   type: STRING
+- mode: NULLABLE
```

Comment:
If I recall correctly the baseline_clients_last_seen query does a select * from the daily table. I think we will also have to include the new fields added to the daily in this schema.

Reply:
Another note, with how the query is structured the column order between the query result in the schema definition is different. I don't think this will break anything in this case, but I also feel like we usually try to maintain the order across the two.